src/python/pants/backend/jvm/artifact.py

```diff
@@ -82,4 +82,7 @@ def __ne__(self, other):
     return not self.__eq__(other)
 
   def __repr__(self):
-    return "{}-{} -> {}".format(self.org, self.name, self.repo)
+    return '{}-{} -> {}'.format(self.org, self.name, self.repo)
+
+  def __str__(self):
```

Comment:
I don't see an edit in here to add `__str__` for python requirements objects. Perhaps you already checked and they are fine?

Reply:
I didn't check, because python libraries never set the `exportable` label, so the mechanism in listtargets never applied to them to begin with. But I guess it will work now, because the test is just for the presence of `provides`. So good call! Added a `__str__` to PythonRequirement.

Reply:
Any other exportable target types to think of?

Reply:
I searched for targets with a provides arg, the only other ones were confluence page targets, so I added `__str__` appropriately there too.